aws/templates/solution/solution_ecs.ftl

```diff
@@ -53,6 +53,47 @@
             name=formatComponentLogGroupName(tier, component) /]
     [/#if]
 
+    [#if deploymentSubsetRequired("efs", true) && 
+            ecs.ClusterWideStorage?has_content && ecs.ClusterWideStorage ]
+
+        [#assign ecsEFSVolumeId = formatEFSId(tier, component)]
+        [#assign ecsEFSVolumeName = formatName( tier, component, "efs")]
+        [#assign ecsEFSSecurityGroupId = formatComponentSecurityGroupId( tier, component,"efs")]
+        [#assign ecsEFSIngressSecurityGroupId = formatDependentSecurityGroupIngressId(ecsEFSSecurityGroupId) ]
+
+        [@createComponentSecurityGroup
+            mode=listMode
+            tier=tier
+            component=component 
+            extensions="efs"
+            /]
+        
+        [@createSecurityGroupIngress
```

Comment:
DO we need this? Maybe ingress rules added when the efs is mounted - so by defualt the efs itself has no ingress rules

Reply:
Two security group components are added, one which creates a security group that controls the EFS eni interfaces and the Ingress update is for the ECS cluster security group. 

Since the two are created as part of the same component then they are added together.

Reply:
ok - my bad